src/backend/utils/systeminfo/index.ts

```diff
@@ -107,12 +111,14 @@ async function getSystemInfo(cache = true): Promise<SystemInformation> {
     },
     steamDeckInfo: deckInfo,
     isFlatpak: !!process.env.FLATPAK_ID,
+    isNixPkg: Boolean(process.env.NIX_PKG),
```

Comment:
Not sure how I feel about having this in the system info, considering the package isn't official in the first place. Perhaps remove it here & just check for `process.env.NIX_PKG` in your change to `main.ts`?